hub/cli/commands.py

```diff
@@ -0,0 +1,19 @@
+import click
+from hub.cli.auth import login, logout, register
+from hub import __version__
+
+
+@click.group()
+@click.version_option(__version__, message="%(prog)s %(version)s")
```

Comment:
Maybe it would be convenient to also have short versions of commands like -v or -h.